scripts/build-code-sidecar.sh

```
@@ -0,0 +1,216 @@
+resources_dir="${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}"
+binary_dir="${resources_dir}/bin"
+client_dir="${resources_dir}/code-sidecar/client"
+monitor_dir="${resources_dir}/code-sidecar/resource-monitor"
+mkdir -p "$binary_dir" "$client_dir" "$monitor_dir"
+rsync -a "$output_binary" "${binary_dir}/${BINARY_NAME}"
+rsync -a --delete --exclude '*.map' "${PACKAGE_DIST}/client/" "$client_dir/"
+# The desktop client only uses the native bridge for its local environment.
+# Remove the hosted-login and telemetry configuration inherited from the
+# pinned web bundle so the renderer has no embedded credential or relay path.
+find "$client_dir" -type f -name '*.js' -print0 \
+  | xargs -0 /usr/bin/perl -0777pi -e 'for my $key (qw(VITE_CLERK_CLI_OAUTH_CLIENT_ID VITE_CLERK_JWT_TEMPLATE VITE_CLERK_PUBLISHABLE_KEY VITE_RELAY_OTLP_TRACES_DATASET VITE_RELAY_OTLP_TRACES_TOKEN VITE_RELAY_OTLP_TRACES_URL VITE_T3CODE_RELAY_URL)) { if (/\Q$key\E:`([^`]*)`/) { my $value = $1; s{`\Q$value\E`}{``}g if length $value; } }'
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)